IBAnimatable/AnimatablePresentatedViewController.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  Created by Tom Baranes on 16/07/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class AnimatablePresentedViewController: UIViewController, PresentationDesignable {
```

Comment:
Using `AnimatablePresentedViewController` as an API is a little bit weird for me. I may confuse you since I used `TransitionPresentedViewController` in the demo app.  Becuase when we use `Present`, `Present` transitions, there also have `PresentedViewController`. How about `AnimatableModalViewController` or other similar? On iPad, sometimes, we use Popover, Some I am not sure `Modal` is a good name for both iPhone or iPad? How do you think?